src/app/calendar/connector.cpp

```diff
@@ -0,0 +1,10 @@
+#include "connector.hpp"
+
+Connector::Connector(const std::chrono::year_month_day& ymd, const std::shared_ptr<Room>& room,
+                     const std::shared_ptr<Doctor>& doctor)
+    : ymd_{ymd}, room_{room}, doctor_{doctor} {};
```

Comment:
Shall we consider situation where we do not know yet if doctor will be available on the date, but we can reserve the room on that day and time?

Reply:
I think there would be no such situation, why would we store information on a visit to a particular room if we don't know which doctor is going to be taking these visits?

Reply:
Refer to reserve queue, solved